src/packages/emmett-mongodb/src/eventStore/mongoDBEventStore.ts

```diff
@@ -110,31 +163,34 @@ export class MongoDBEventStore implements EventStore<number> {
   async appendToStream<EventType extends Event>(
     streamName: StreamName,
     events: EventType[],
-    options?: AppendToStreamOptions<number> & {
-      /**
-       * These will be ran after a the events have been successfully appended to
-       * the stream. `appendToStream` will return after every projection is completed.
-       */
-      projections?: Array<Projection<EventType>>;
-    },
-  ): Promise<AppendToStreamResult<number>> {
-    let stream = await this.collection.findOne({
-      streamName: { $eq: streamName },
-    });
-    let currentStreamPosition = stream?.events.length ?? 0;
+    options?: AppendToStreamOptions,
+  ): Promise<AppendToStreamResult> {
+    let stream = await this.collection.findOne(
```

Comment:
**CONCERN:** Is it intentional, that kept here the reading the document? We discussed in the other thread, not to do it unles there are inline projections registered.

Reply:
Removed